src/components/HomeView.jsx

```diff
@@ -0,0 +1,12 @@
+import { FeedView } from "./FeedView"
+import { ArticleView } from "./ArticleView"
+import { Routes, Route } from 'react-router'
+
+export const HomeView = () => {
+    return (
+        <Routes>
+            <Route path='/' element={<FeedView />}></Route>
+            <Route path='/articles/:article_id' element={<ArticleView />}></Route>
+        </Routes>
```

Comment:
Unsure what the aim was here but you should only have one route per endpoint, I think this stuff in here is unnecessary.